Xamarin.Forms.Core/GestureType.cs

```diff
@@ -0,0 +1,20 @@
+namespace Xamarin.Forms
+{
+	public enum GestureType
```

Comment:
I don't get the meaning of this

Reply:
These are the directions of finger movements.

Reply:
Could this be `GestureDirection` and I wonder if we should be more explicit like LeftToRight, RightToLeft?

Reply:
done